### DISCUSSION TITLE
Is there a way to vary the colors?

### DISCUSSION THREAD

**Discussion by PaekHyun** [MMRIZE/MMM-CalendarExt3#157] [Q&A] [answered]
First of all, thank you for developing the calendar module.
MMM-CalendarExt3Timeline and MMM-CalendarExt3
I am using Module.

But various colors depending on the date? Or I want to do it randomly.
Is this method possible?

If possible, what should I do?

**Comment by eouia** [comment, marked as answer]
Are you Korean?
한국분이신 것 같아, 한글로 답할게요.
config에서 `evnetTransformer` callback 함수를 사용해서 event의 속성들을 바꾸실 수 있습니다.
링크의 예제는 랜덤하게 색상을 바꾸는 건 아니지만, 특정 타이틀 이벤트의 색상을 바꾸시는 예제에요.
https://github.com/MMRIZE/MMM-CalendarExt3?tab=readme-ov-file#transforming

**Comment by dthanh267** [comment]
it's not working for me, do i put it in config.js or ccs?

**Reply by eouia** [reply]
Sorry for not getting back to you sooner, I missed this question. (If possible, post a new topic)
`eventTransformer` should be placed inside of the config of module (In CX3 config section in your `config/config.js`)
